.gitignore

```diff
@@ -10,13 +10,7 @@
 # Mobile Tools for Java (J2ME)
 .mtj.tmp/
 
-.idea/workspace.xml
-.idea/dictionaries
-.idea/misc.xml
-.idea/libraries
-.idea/uiDesigner.xml
-.idea/kotlinc.xml
-.idea/modules.xml
+.idea/
```

Comment:
Why's this part of it

Reply:
I haven't committed the project files yet as I had added it as a sub module to my Paper project, but intent was to commit the project files so we don't really need this add here @me4502

Reply:
Ah okay, I added it because there were about 30 project files that weren’t ignored. I’ll remove this then

Reply:
I REALLY dislike including the `.idea/` directory. I don't give a rat's ass that these files are "meant to be shared." I'm much happier for you to leave this in.

Reply:
I'm keeping as is for now as I'm not sure what's wanted haha...